mandatory/3-function-output.js

```diff
@@ -1,16 +1,19 @@
 // Add comments to explain what this function does. You're meant to use Google!
+// This function will return a random number in a ranfe of up to number 10.
```

Comment:
```suggestion
// This function will return a random number in a range of up to number 10.
```

Reply:
Noticed a small typo in your answer. Please find suggested change.

Reply:
Thank you for noticing this Christina, I will fix it later today :)